src/components/AddRemoveFavourite.tsx

```diff
@@ -22,26 +24,17 @@ interface AddRemoveFavouriteProps {
   onClick: () => void;
 }
 
-const AddRemoveFavourite: FC<AddRemoveFavouriteProps> = ({
-  isFavourite,
-  onClick,
-}) => {
+const AddRemoveFavourite: FC<AddRemoveFavouriteProps> = ({ isFavourite, onClick }) => {
   return (
     <StyledButton onClick={onClick}>
       {isFavourite ? (
         <>
-          <img
-            src={require('../../../assets/icons/heart_full.png')}
-            alt="ikona_serce_puste"
-          />
+          <img src={FullHeart} alt="ikona_serce_wypełnione" />
```

Comment:
Maybe with this change we can also consider removing underscores from alt text. @MariaRukat ?

Reply:
removed